packages/relay/src/lib/eth.ts

```diff
@@ -1404,12 +1404,27 @@ export class EthImpl implements Eth {
 
     let txSubmitted = false;
     try {
-      const contractExecuteResponse = await this.hapiService
-        .getSDKClient()
-        .submitEthereumTransaction(transactionBuffer, EthImpl.ethSendRawTransaction, requestIdPrefix);
+      const contractExecuteResponse = await this.sendRawTransactionWithRetry(
```

Comment:
Q: Why are we retrying for sendRawTransaction vs leaving it on user client logic?
The challenge is here is there's balance implication to both the sender and the host.